src/nvim/api/win_config.c

```diff
@@ -958,6 +958,12 @@ static bool parse_float_config(Dict(float_config) *config, FloatConfig *fconfig,
     has_relative = true;
     fconfig->external = false;
     if (fconfig->relative == kFloatRelativeWindow) {
+      if (reconf) {
+        win_T *target_win = find_window_by_handle(config->win, err);
+        if (!target_win || target_win == wp) {
+          return false;
```

Comment:
I don't think this `return` is correct. This causes the other keys in the config to be silently ignored.

Reply:
when win no target win or target is itself the other keys still need parse ? this set should execute  right ?

Reply:
That will not solve the usage mentioned in #24129